src/main/java/core/db/DBUpdater.java

```diff
@@ -228,9 +213,33 @@ private void updateDBv300(int DBVersion, int version) throws SQLException {
 		if (!columnExistsInTable("RATINGINDIRECTSETPIECESDEF", MatchDetailsTable.TABLENAME)) {
 			m_clJDBCAdapter.executeUpdate("ALTER TABLE MATCHDETAILS ADD COLUMN RATINGINDIRECTSETPIECESDEF INTEGER");
 		}
+
+		//store FirstName, Nickname  into Playertable
+		if (!columnExistsInTable("FirstName", SpielerTable.TABLENAME)) {
+			m_clJDBCAdapter.executeUpdate("ALTER TABLE SPIELER ADD COLUMN FirstName VARCHAR (100)");
+			m_clJDBCAdapter.executeUpdate("ALTER TABLE SPIELER ADD COLUMN NickName VARCHAR (100)");
+			m_clJDBCAdapter.executeUpdate("ALTER TABLE SPIELER ALTER COLUMN Name RENAME TO LastName");
```

Comment:
For historic data, the full name will be the last name.  Is that an issue?  For now the only problem I have noticed is that until I download data on an existing db, full names are displayed in Squad and Lineup, and after, only first letter of first name + last name.

Reply:
> For historic data, the full name will be the last name. Is that an issue? 

I don't think it will be too much an issue. But anyway there is no real choice here, this is also because of this (the impossibility to always guess first name, nickname and last name from splitting the full name ) that I made that change in first place.

Reply:
>  (the impossibility to always guess first name, nickname and last name from splitting the full name 
Exactly, hence my earlier comment about “re-processing.”  –  but I think you're right, it's probably not a big deal. 👍